ansible/roles/demo-certs/tasks/main.yml

```diff
@@ -0,0 +1,25 @@
+# creates a local CA for demo purposes, should not be used in prod
```

Comment:
[question] if `demo_cert` is the **role** needed to handle certs (local CA or ext provided) why is it called `demo_*`?

I think I understand that we also include the local CA creation but it also includes the configuration settings (inside **install-certs**) which will be needed for production users.

Reply:
I want to strongly discourage using this module in production, so I've given it a name that will cause people to really strongly consider whether they want to use it vs simply handling their own certs. 

The install-certs import is actually incorrect and I've removed it in the other PR -- I know it's confusing, but I didn't want to clobber this one with linter changes.